### PR TITLE
#95 Added Docker Hub search Query Engine

### PR DESCRIPTION
**Pull request by ritishbhardwaj**
## What is this Pull Request About?

To add search engine query for Docker Hub 
useful in DevOps

## What will this Pull Request Affect?
will show the respective or required registry for containerization

# Any additional information?

### CODE REVIEW

**Comment by ritishbhardwaj** [comment]
@ArjunSharda 
kindly merge this PR

File: src/searchor/__init__.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Engine(Enum):
     CrunchyrollBeta = "https://beta.crunchyroll.com/search?q={query}"
     DuckDuckGo = "https://www.duckduckgo.com/?q={query}"
     Dictionary = "https://www.dictionary.com/browse/{query}"
+    DockerHub= "https://hub.docker.com/search?q={query}"
```
Comment (ArjunSharda) [inline src/searchor/__init__.py:22]:
```suggestion
    DockerHub = "https://hub.docker.com/search?q={query}"
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
I've left a suggestion. Can you please commit it and then in the next update I can merge it?

**Comment by ritishbhardwaj** [comment]
@ArjunSharda  
sure, and please checkout i have made the change

**Review by ArjunSharda — APPROVED** [review]
LGTM! Will merge in the next update.

**Pull request merged**